packs/chef/actions/chef_install.py

```diff
@@ -20,9 +19,10 @@ class ChefInstaller(object):
         ('-d', '--download_path', {})
     ]
 
-    def install(self, install_method, options):
+    @staticmethod
+    def install(install_method, options):
         exit_code = 0
-        installer = getattr(ChefInstaller, install_method.capitalize())(options)
+        installer = getattr(sys.modules['lib'], install_method.capitalize())(options)
```

Comment:
Looking at the code - it doesn't look like the previous code ever worked since the " from omnibus .." line is commented.